app/jobs/build_job.rb

```diff
@@ -82,6 +84,16 @@ def set_status(status)
     end
   end
 
+  def set_started_at
```

Comment:
How do you feel about making these methods on the model versus methods in the build job?